webapp/src/Controller/Jury/JudgehostController.php

```diff
@@ -357,7 +358,7 @@ public function deleteAction(Request $request, string $hostname): Response
      */
     public function activateAction(RouterInterface $router, Request $request, string $hostname): RedirectResponse
     {
-        $judgehost = $this->em->getRepository(Judgehost::class)->find($hostname);
+        $judgehost = $this->em->getRepository(Judgehost::class)->findOneBy(['hostname' => $hostname]);
```

Comment:
This UI action and the one below could just use the ID as the parameter as would other UI parts. Internally, use judgehostid as the reference where we can.